File: integration-tests/mongodb-panache/src/main/java/io/quarkus/it/mongodb/panache/reactive/person/ReactivePersonRepository.java
package io.quarkus.it.mongodb.panache.reactive.person;

import jakarta.enterprise.context.ApplicationScoped;

import io.quarkus.it.mongodb.panache.person.Person;
import io.quarkus.mongodb.panache.reactive.ReactivePanacheMongoRepositoryBase;

@ApplicationScoped
public class ReactivePersonRepository implements ReactivePanacheMongoRepositoryBase<Person, Long> {
}

File: integration-tests/mongodb-panache-kotlin/src/main/kotlin/io/quarkus/it/mongodb/panache/reactive/person/ReactivePersonRepository.kt
package io.quarkus.it.mongodb.panache.reactive.person

import io.quarkus.it.mongodb.panache.person.Person
import io.quarkus.mongodb.panache.kotlin.reactive.ReactivePanacheMongoRepositoryBase
import jakarta.enterprise.context.ApplicationScoped

@ApplicationScoped
class ReactivePersonRepository : ReactivePanacheMongoRepositoryBase<Person, Long>
